tests/backup/backup_basic_test.go

```diff
@@ -56,6 +57,7 @@ var _ = AfterSuite(func() {
 	if wantAllAfterSuiteActions || wantAfterSuiteValidateCleanup {
 		ValidateCleanup()
 	}
+	defer dash.TestCaseEnd()
```

Comment:
Defer should be the first statement,
Also add TestSetEnd

Reply:
done!!